contracts/VoucherKernel.sol

```diff
@@ -13,7 +13,7 @@ import "./interfaces/IVouchers.sol";
 import "./interfaces/IVoucherKernel.sol";
 import {PaymentMethod, VoucherState, VoucherStatus, isStateCommitted, isStateRedemptionSigned, isStateRefunded, isStateExpired, isStatus, determineStatus} from "./UsingHelpers.sol";
 
-//preparing for ERC-1066, ERC-1444, EIP-838
+//preparing for ERC-1066, ERC-1444
```

Comment:
Please just remove this whole line. We're no planning to implement any of these ERCs.

Reply:
removed.